Recast/HomeViewController.swift

```diff
@@ -0,0 +1,219 @@
+//
+//  HomeViewController.swift
+//  Recast
+//
+//  Created by Jaewon Sim on 9/19/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+struct DummyPodcastSeries {
+    let image: UIImage
+    let title: String
+    let date: String
+    let duration: Int
+    let timeLeft: Int
+
+}
+
+class HomeViewController: UIViewController {
+
+    //dummy structs for testing
+    let dummyContinue1 = DummyPodcastSeries(image: #imageLiteral(resourceName: "series_img_1"), title: "110: Asking All the Questions", date: "September 13, 2018", duration: 58, timeLeft: 10)
+    let dummyContinue2 = DummyPodcastSeries(image: #imageLiteral(resourceName: "series_img_2"), title: "Dummy Podcast 2", date: "November 1st, 2018", duration: 10, timeLeft: 29)
+
+    var homeTableView: UITableView!
+    // Maps section/row index to the appropriate header type
+    let headerTypes: [Int: HomeSectionType] = [
+        0: .continueListening,
+        1: .yourFavorites,
+        2: .browseTopics
+    ]
+
+    /// Array of podcasts for the "continue listening" section
+    var continuePodcasts: [DummyPodcastSeries] = [] {
+        didSet {
+            guard let cv = view.viewWithTag(100) as? UICollectionView else { return }
```

Comment:
I'm not sure if we want to use straight ints for each collectionview, make these potentially into an enum that encapsulates the section as well as the collectionview tag

Reply:
^^ add an enum for collectionViewTypes with a raw value for a tag

Reply:
ah wait then you can't switch on `collectionViewTypes[collectionView.tag]` in your CV datasource